force-app/main/default/classes/BDI_ContactService.cls

```diff
@@ -1146,6 +1258,15 @@ public with sharing class BDI_ContactService {
         } private set;
     }
 
+    /*******************************************************************************************************
+    * @description whether the Duplicate Management matching option is in the contact matching rules
+    */
+    private boolean isDuplicateManagement {
+        get {
+            return (bdi.diSettings.Contact_Matching_Rule__c == System.Label.bdiContactMatchDupeMgmt ? true : false);
```

Comment:
Is there no longer a way to set Matching Rules per batch? I assume no, but I know there's a field on the DI Batch object.

Reply:
There is, the batch would be converted into Data import settings overriding the default ones if it had different matching criteria.  But we should definitely test it with the new matching method.